concrete_datastore/concrete/management/commands/reset_password.py

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+import random
+import string
+
+from django.apps import apps
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+
+class Command(BaseCommand):
+    help = 'Reset password'
+
+    def add_arguments(self, parser):
+        parser.add_argument('email', type=str, help='user email')
+
+    def handle(self, *args, **options):
+        concrete = apps.get_app_config('concrete')
+        EmailModel = concrete.models['email']
+        UserModel = get_user_model()
+        email = options['email'].lower()
+
+        try:
+            user = UserModel.objects.get(email=email)
+        except UserModel.DoesNotExist:
+            raise CommandError(f"This email: {email} does not exist")
+
+        password = ''.join(
+            random.choice(string.ascii_letters + string.digits)  # nosec
+            for _ in range(24)
+        )
```

Comment:
These are the same lines as the lines 26 -> 29 of the file concrete_datastore/concrete/management/commands/create_superuser_with_email.py

Maybe you should create a method 
```python
def generate_random_password(
    length=24, chars=string.ascii_letters + string.digits
):
    return ''.join(random.choice(chars) for _ in range(length))
```